internal/fixtures/cookie

```diff
@@ -0,0 +1 @@
+clsession=MTUzMTkyMzI0NHxEdi1CQkFFQ180SUFBUkFCRUFBQVJ2LUNBQUVHYzNSeWFXNW5EQTRBREdOc2MyVnpjMmx2Ymw5cFpBWnpkSEpwYm1jTUlnQWdNR016TWpobVptWmhaVFkxTkRNd05UaGpNelF3WXpZMFlqTTJNVGM1WkRBPXzDcp8k2N2ylrXA_3AQZlqiYpQqJIZOlpnh9XkQnj2HFQ==; Path=/; Expires=Fri, 17 Aug 2018 14:14:04 GMT; Max-Age=2592000
```

Comment:
Do we have to worry about this expiring?

Reply:
No, it's merely a fixture for deserializing, not actual authentication.